tests/test_download.py

```diff
@@ -125,38 +84,29 @@ def mock_api_call(self, operation_name, api_params):
         assert response["Access-Control-Allow-Methods"] == "GET"
 
 
-def test_client_legacy_product_prefix(client, gcsmock, metricsmock):
-    reload_downloaders("https://storage.googleapis.example.com/private/prefix/")
-
-    mock_bucket = gcsmock.MockBucket()
-
-    gcsmock.get_bucket = lambda name: mock_bucket
-
-    def mock_get_blob(key):
-        if key == "prefix/v0/xul.pdb/44E4EC8C2F41492B9369D6B9A059577C2/xul.sym":
-            # As if it exists happily
-            blob = gcsmock.mock_blob_factory(key)
-            blob.public_url = "https://googleapis.example.com/private/" + key
-            return blob
-
-        raise NotImplementedError(key)
+def test_client_legacy_product_prefix(client, botomock):
```

Comment:
In other cases like this, a GCS-only test was later added, so the test needs to be converted to use ``botomock`` and a ``mock_api_call`` inner function.

I also noticed that ``metricsmock`` was added as an unused fixture. I've removed these as I found them.